custom/elements.json

```
@@ -577,39 +577,46 @@
           "bias",
           "divisor",
           "edgeMode",
-          "in",
+          {
+            "in": "in1",
+            "kernelUnitLength": "kernelUnitLengthX",
+            "order": "orderX"
+          },
           "kernelMatrix",
```

Question: This break the alphabetical order, shouldn't we do the following?

```suggestion
          {"in": "in1"},
          "kernelMatrix",
          {
            "kernelUnitLength": "kernelUnitLengthX",
            "order": "orderX"
          },
```

---

If you look at what's already in the file, then objects are always combined and things aren't always in alphabetical order. I don't think it's needed.